dojo/pipeline.py

```diff
@@ -1,14 +1,33 @@
+from django.conf import settings
+
+
 def social_uid(backend, details, response, *args, **kwargs):
-    uid = backend.get_user_id(details, response)
-    # Used for most backends
-    if uid:
-        return {'uid': uid}
-    # Until OKTA PR in social-core is merged
-    # This modified way needs to work
+    if settings.AZUREAD_TENANT_OAUTH2_ENABLED:
```

Comment:
This breaks OKTA if Azure is enabled as well. A new user can be created but cannot login.

Reply:
@Maffooch I don't see a case where both would be enabled.

Reply:
I will try to get an okta dev account so I create an if/else scenario that does not break it